tui/tests/test_snapshots.py

```
@@ -0,0 +1,77 @@
+async def _stabilise_insights(pilot) -> None:
+    """Pin TZ to UTC before rendering the Insights tab.
+
+    The ``peak_hour`` metric is computed from UTC hours offset by the local
+    timezone, so different contributor machines produce different snapshots.
+    Pinning ``TZ=UTC`` ensures the golden file is reproducible everywhere.
+    """
+    os.environ["TZ"] = "UTC"
+    time.tzset()
+
+
+@pytest.mark.parametrize("tab_id, keys", TAB_SPECS, ids=[t[0] for t in TAB_SPECS])
```

Ruff (0.15.4)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)